docs/source/customization/mcp-tools.md

```
@@ -338,8 +342,17 @@ for protocol details.
-  AIQ sign-in and does not authorize the upstream MCP server.
+- Use a nested `per_user_auth` block (the `PerUserAuthConfig` object, not a bare boolean) for upstream
+  MCP OAuth; `requires_auth: true` only gates a source on AIQ sign-in and does not authorize the
+  upstream MCP server:
+
+  ```yaml
+  per_user_auth:
+    required: true                     # gate job submission until the user connects
+    provider: google                   # provider identifier
+    mcp_server_id: gdrive              # key used to look up the user's token in NAT token storage
+    auth_provider: mcp_oauth2_gdrive   # the `authentication` (mcp_oauth2) provider for this source
+  ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)